src/notes_handlers.py

```diff
@@ -1,14 +1,18 @@
 from src.constants import INFO, UPDATED
 from src.models.Note import Note
 from src.models.NoteBook import NoteBook
-from src.decorators import input_error
+from src.decorators import collect_note_details, input_error
```

Comment:
collect_note_details does not exist

Reply:
yeah, i have removed it. Instead we go with "parse_input_add_note"

Reply:
Done.